faqs/galaxy/datatypes_fastq_format.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to format fastq data for tools that require .fastqsanger format
+area: datatypes
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+- Most tools that accept FASTQ data expect it to be in a specific FASTQ version: .fastqsanger. The .fastqsanger datatype must be assigned to each FASTQ dataset.
+
+Inorder to do that:
+
+- Watch the **[FASTQ Prep Illumina](http://vimeo.com/galaxyproject/fastqprep)** video for a complete walk-through.
+- Run **FastQC** first to assess the type.
+    - Run **FASTQ Groomer** if the data needs to have the quality scores rescaled.
+    - If you are certain that the quality scores are already scaled to Sanger Phred+33 (the result of an Illumina 1.8+ pipeline), the datatype ".fastqsanger" can be directly assinged. Click on the pencil icon {% icon galaxy-pencil %} to reach the **Edit Attributes** form. In the center panel, click on the "Datatype" tab, enter the datatype ".fastqsanger", and save. Metadata will assign, then the dataset can be used.
+- Run **FastQC** again on the entire dataset *if any changes were made to the quality scores* for QA.
+
+**Other tips**
+
+- If you are not sure what type of FASTQ data you have (maybe it is not Illumina?), see the help directly on the **FASTQ Groomer** tool for information about types.
+    - For _Illumina_, first run **FastQC** on a sample of your data ([how to read the full report](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/)). The output report will note the quality score type interpreted by the tool. If not ".fastqsanger", run **FASTQ Groomer** on the entire dataset. If '.fastqsanger", just assign the datatype.
```

Comment:
For consistency, I think that ".fastqsanger" should be `.fastqsanger`, but please check other FAQs that mention file formats